src/Http/Routing/src/Builder/OpenApiDelegateEndpointConventionBuilderExtensions.cs

```diff
@@ -3,6 +3,8 @@
 
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http.Metadata;
+using Microsoft.AspNetCore.Mvc;
```

Comment:
This was the entire point of this change right? 
```suggestion
```

Reply:
The `ProblemDetails` class is still defined in the MVC namespace (although it is in the Http assembly).

Reply:
Ugh, ok 😢